packages/config/README.md

```diff
@@ -0,0 +1,71 @@
+# Collection of Base Configurations for Monorepo
+
+The package name is `@cloudoperators/juno-config`.
+
+- Typescript
+- ESlint
+
+## Typescript
+
+This package provides a base TypeScript configuration to be used across multiple packages in a monorepo setup. It ensures consistent TypeScript settings and simplifies the TypeScript setup process for each package.
+
+## Usage
+
+You can use this configuration as a starting point in any package or app within your monorepo using TurboRepo.
+
+```javascript
+packages / package - a / src / tsconfig.json
+eslint.config.mjs
+```
+
+1. **Create a `tsconfig.json` in your package or app:**
+
+```json
+{
+  "extends": "@cloudoperators/juno-config/typescript/base.json",
+  "compilerOptions": {
+    "outDir": "./build",
+    "rootDir": "./src"
+  },
+  "include": ["src"]
+}
+```
+
+## ESLint
+
+This package provides also a temporary ESLint configuration specifically for TypeScript to be used across multiple packages in a monorepo setup.
```

Comment:
What does "temporary" mean in this context?

Reply:
Cause we need two configs one for typescript one for old javascript yet. But I will delete word temporary